cluster/src/main/java/com/hartwig/pipeline/PipelineMain.java

```diff
@@ -46,7 +46,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(PipelineMain.class);
 
-    public PipelineState start(Arguments arguments) {
+    public PipelineState start(Arguments arguments) throws Exception {
```

Comment:
Why does this method throw now?

Reply:
That was unintentional so feel free to revert!